pkg/controllers/machinehealthcheck/machinehealthcheck_controller_test.go

```diff
@@ -322,6 +322,33 @@ func TestHasMachineSetOwner(t *testing.T) {
 
 }
 
+func TestHasRemediationDisabledAnnotation(t *testing.T) {
```

Comment:
I do not sure what value this unittest has, I prefer to have unittest to check that reconcile exit with `reconcile.Result{}, nil` when the annotation exists

Reply:
If I will test the returned nil I will not able to distinguish between a call with annotation to a call without annotation , both return nill ... Please advice